### DISCUSSION TITLE
is there any github student pack WordPress supported hosting packages?

### DISCUSSION THREAD

**Discussion by pkofficial46** [community/community#132096] [GitHub Education] [answered]
WordPress ?

**Comment by nadeeshaofficial1** [comment, marked as answer]
yes. Heroku and DigitalOcean. you can install WordPress on it !.

**Reply by champika1977** [reply]
Ok

**Comment by nadeeshaofficial1** [comment]
also, you can use Microsoft Azure free credits!

**Comment by rajeev77gagan** [comment]
Myself Rajeev kumar

**Comment by ChandruMuthuswamy** [comment]
The GitHub Student Developer Pack offers various web hosting options that support WordPress. Here are some of the options available:

DigitalOcean:

DigitalOcean offers $200 in credits to students. This can be used to deploy WordPress through their 1-Click Apps feature, which makes it straightforward to set up a WordPress site on a DigitalOcean droplet.
Microsoft Azure:

Azure provides $100 in credits for students, which can be used to host web applications, including WordPress. However, it's important to note that the credits cannot be applied directly to Azure Marketplace offers for WordPress. Instead, you can use free trials or free tier plans offered by Azure Marketplace partners to host your WordPress site.
Namecheap:

Namecheap offers a free domain for a year and discounts on web hosting, which can be used to host a WordPress site.
These options provide various ways to host a WordPress site while leveraging the benefits provided by the GitHub Student Developer Pack. For more detailed information and to explore all the offers, you can visit the [GitHub Student Developer Pack page](https://education.github.com/pack).

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @pkofficial46!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Education category. I've gone ahead and moved it for you. Good luck!
